compatibility.md

```diff
@@ -0,0 +1,59 @@
+# WordPress compatibility
+
+The relationship between WordPress and PHP is very close, and it is very important that the versions of both technologies match for proper operation. These matrix are focused on information for hosting companies and sysadmins.
+
+## WordPress and PHP versions matrix
+
+WordPress | PHP min version | PHP max version | Launch date | Supported PHP
+------------ | ------------- | ------------- | ------------- | -------------
+WordPress 5.6 | PHP 5.6.20 | PHP 8.0* | 2020-12-08* | PHP 7.3 - 8.0*
+WordPress 5.5 | PHP 5.6.20 | PHP 7.4 | 2020-08-11 | PHP 7.2 - 7.4
+WordPress 5.4 | PHP 5.6.20 | PHP 7.4 | 2020-03-31 | PHP 7.2 - 7.4
+WordPress 5.3 | PHP 5.6.20 | PHP 7.4 | 2019-11-12 | PHP 7.1 - 7.3
+WordPress 5.2 | PHP 5.6.20 | PHP 7.3 | 2019-05-07 | PHP 7.1 - 7.3
+WordPress 5.1 | PHP 5.2.4 | PHP 7.3 | 2019-02-21 | PHP 7.1 - 7.3
+WordPress 5.0 | PHP 5.2.4 | PHP 7.3 | 2018-12-06 | PHP 5.6 - 7.2
+WordPress 4.9 | PHP 5.2.4 | PHP 7.2 | 2017-11-15 | PHP 5.6 - 7.1
+WordPress 4.8 | PHP 5.2.4 | PHP 7.1 | 2017-06-08 | PHP 5.6 - 7.1
+WordPress 4.7 | PHP 5.2.4 | PHP 7.1 | 2016-12-06 | PHP 5.6 - 7.0
+WordPress 4.6 | PHP 5.2.4 | PHP 7.0 | 2016-08-16 | PHP 5.5 - 7.0
+WordPress 4.5 | PHP 5.2.4 | PHP 7.0 | 2016-04-12 | PHP 5.5 - 7.0
+WordPress 4.4 | PHP 5.2.4 | PHP 7.0 | 2015-12-08 | PHP 5.5 - 5.6
+WordPress 4.3 | PHP 5.2.4 | PHP 5.6 | 2015-08-18 | PHP 5.4 - 5.6
+WordPress 4.2 | PHP 5.2.4 | PHP 5.6 | 2015-04-23 | PHP 5.4 - 5.6
+WordPress 4.1 | PHP 5.2.4 | PHP 5.6 | 2014-12-17 | PHP 5.4 - 5.6
+WordPress 4.0 | PHP 5.2.4 | PHP 5.5 | 2014-09-04 | PHP 5.4 - 5.5
+WordPress 3.9 | PHP 5.2.4 | PHP 5.5 | 2014-04-16 | PHP 5.3 - 5.5
+WordPress 3.8 | PHP 5.2.4 | PHP 5.5 | 2013-12-12 | PHP 5.3 - 5.4
+WordPress 3.7 | PHP 5.2.4 | PHP 5.5 | 2013-10-24 | PHP 5.3 - 5.4
+
+* WordPress 5.6 and PHP 8.0 are expected to be launched at the end of 2020.
```

Comment:
Outdated information, should be removed?

Reply:
This page is right now in "stand-by" because there are som other pages about... I love this page to be in our handbook, but core has it's own...

also, yes, right now it's outdated, but not published.